src/dependencies/DependencyMap.js

```diff
@@ -135,12 +135,13 @@ export function calculateForDispatch(
 
 export function calculateForPropsChange(
   dependencies: DependencyMap,
-  props: Object
+  props: Object,
+  state: ?Object
 ): Object {
   return oFilterMap(
     dependencies,
     (dep) => dep.deref && dep.deref.length > 0,
-    (dep) => calculate(dep, props)
+    (dep) => calculate(dep, props, state)
```

Comment:
@marcneuwirth sorry I was thinking too hard... `caclulate` does the arity check, it was good the way you had it. Changed it back 8906900.